extension/src/test/suite/helper.ts

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
is this file used anywhere?

Reply:
Yes, I am trying to move more and more public functions used for unit test here.
I am using this module during the test case writting.

Reply:
I don't see this file imported in other file, and I don't see "sleep" or "princ" out of this file. Is there code on your machine that is not yet uploaded?

Reply:
As I said during my test cases writting I have imported and used this file to quick log messages I needed.
But once git push I removed these usages.